source/libsass.html.haml

```diff
@@ -1,10 +1,10 @@
 ---
 title: LibSass
 introduction: >
-  We want everyone to enjoy Sass, no matter what language they use. Sass was
-  originally written in Ruby. LibSass is a C/C++ port of the Sass engine. The
-  point is to be simple, faster, and easy to integrate. Find out more about the
-  project over at [GitHub](http://github.com/sass/libsass).
```

Comment:
is there still a link to the github repo of the project left in the page as you removed that one ?

Reply:
I don't think it's necessary to include it since we are discouraging its use.